File: CPE 102/lab05/Point.java
/**
 * This is an interface, used in CartesianPoint and PolarPoint classes for PointTester
 *
 * @author Michael Elliott
 * @version Project 01
 * @version CPE102-10 (replace the X with your section number)
 * @version Spring 2015
 */


public interface Point
{
	double xCoordinate();
	double yCoordinate();
	double radius();
	double angle();
	double distanceFrom(Point other);
	Point rotate90();
}

File: CPE 101/lab3/object_equality/point.py
# Name: Michael Elliott
# Instructor: Dr. John Campbell
# Section: 17

import utility 

class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def __eq__(self, other):
        return (utility.epsilon_equal(self.x, other.x)) and (utility.epsilon_equal(self.y, other.y))

        

